### ISSUE TITLE
2.03 (Windows): No hashrate with KawPow (4.3 GB DAG) on GTX 1660 Super

### ISSUE THREAD

**Issue by UselessGuru** [sp-hash/TeamBlackMiner#397]
`TeamBlackMiner-v2.03\TBMiner.exe --algo kawpow --hostname eu.mining.prohashing.com --wallet ********************--port 3361 --server-passwd a=kawpow,e=off,n=************,o=****************--api --api-version 1.4 --api-port 4006 --cuda-devices [0]`

No error, no hashrate until **150 seconds after miner launch**

```
00:00:00 [2023-10-10 13:38:14.384]
00:00:00 [2023-10-10 13:38:14.495] OpenCL driver version: 537.42
00:00:00 [2023-10-10 13:38:14.575] Cuda driver version: 12.2
00:00:00 [2023-10-10 13:38:14.575] Cuda runtime API version: 12.2
00:00:00 [2023-10-10 13:38:14.575]
00:00:00 [2023-10-10 13:38:14.575]
00:00:00 [2023-10-10 13:38:14.575]
00:00:00 [2023-10-10 13:38:14.575] Welcome to Team Black Miner 2.03
00:00:00 [2023-10-10 13:38:14.575]
00:00:00 [2023-10-10 13:38:14.575]
00:00:00 [2023-10-10 13:38:14.887] GPU0 NVIDIA GeForce GTX 1660 SUPER 6GB added to miner as Cuda device
00:00:00 [2023-10-10 13:38:14.906] Trying eu.mining.prohashing.com:3361 (kawpow)
00:00:00 [2023-10-10 13:38:14.924] GPU0 XIntensity set to 256
00:00:00 [2023-10-10 13:38:14.924] Api server started at 127.0.0.1:4006
00:00:00 [2023-10-10 13:38:15.009] eu.mining.prohashing.com: Pool responded to subscribe
00:00:00 [2023-10-10 13:38:15.022] GPU0 Starting generating kawpow dag (Epoch: 401)
00:00:00 [2023-10-10 13:38:15.068] eu.mining.prohashing.com: Pool responded successfully to authorize
00:00:00 [2023-10-10 13:38:18.749] GPU0 Generating dag epoch 401
00:00:00 [2023-10-10 13:38:18.756] GPU0 DAG generated in 3.73 seconds (4333565kb)
00:00:01 [2023-10-10 13:39:15.089] prohashing.com (kawpow)    PING: 34ms    DIFFICULTY: 268.44 M    EPOCH: 401
00:00:01 [2023-10-10 13:39:15.089]
00:00:01 [2023-10-10 13:39:15.089] ID      BOARD    TYPE    KERN    XINT    TEMP    FAN     CORE    MEM     WATT
00:00:01 [2023-10-10 13:39:15.089] GPU0    1660S    CUDA    0       0       45/0    43      300     405     33
00:00:01 [2023-10-10 13:39:15.089]                                                                          33
00:00:01 [2023-10-10 13:39:15.089] ID      BOARD    HASHRATE/W    HASHRATE      AVERAGE       SHARES       RATE
00:00:01 [2023-10-10 13:39:15.089] GPU0    1660S    0.00 H/W      0.00 H/s      0.00 H/s      0/0/0        100.00%
00:00:01 [2023-10-10 13:39:15.089]                  0.00 H/W      0.00 H/s      0.00 H/s      0/0/0        100.00%
00:00:01 [2023-10-10 13:39:15.089]
00:00:01 [2023-10-10 13:39:15.089] SHARES PER MINUTE: 0.00                              POOL HASHRATE: 0.00 H/s
00:00:01 [2023-10-10 13:39:15.089]
00:00:01 [2023-10-10 13:40:09.680] Job a14729e7
00:00:02 [2023-10-10 13:40:33.760] Job a1472a65
00:00:02 [2023-10-10 13:40:33.843] prohashing.com (kawpow)    PING: 46ms    DIFFICULTY: 268.44 M    EPOCH: 401
00:00:02 [2023-10-10 13:40:33.843]
00:00:02 [2023-10-10 13:40:33.843] ID      BOARD    TYPE    KERN    XINT    TEMP    FAN     CORE    MEM     WATT
00:00:02 [2023-10-10 13:40:33.843] GPU0    1660S    CUDA    0       0       42/0    42      300     405     22
00:00:02 [2023-10-10 13:40:33.843]                                                                          22
00:00:02 [2023-10-10 13:40:33.843] ID      BOARD    HASHRATE/W    HASHRATE      AVERAGE       SHARES       RATE
00:00:02 [2023-10-10 13:40:33.843] GPU0    1660S    0.00 H/W      0.00 H/s      0.00 H/s      0/0/0        100.00%
00:00:02 [2023-10-10 13:40:33.844]                  0.00 H/W      0.00 H/s      0.00 H/s      0/0/0        100.00%
00:00:02 [2023-10-10 13:40:33.844]
00:00:02 [2023-10-10 13:40:33.844] SHARES PER MINUTE: 0.00                              POOL HASHRATE: 0.00 H/s
00:00:02 [2023-10-10 13:40:33.844]
```

**Comment by sp-hash** [comment]
fixed in 2.04

**Comment by UselessGuru** [comment]
Confirmed - issue is solved.
